test/index-test.js

```diff
@@ -4,10 +4,12 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import jsdom from 'mocha-jsdom'
 import FontAwesome from '../src'
+import styles from '../src/react-fontawesome.css'
```

Comment:
As you want. I did it in the latest commit, but these tests seems to me to be super fragile. :/